README.md

```diff
@@ -1,76 +1,121 @@
 # Records Mover - mvrec
 
-Records Mover is a command-line tool and Python library you can
+Records mover is a command-line tool and Python library you can
 use to move relational data from one place to another.
 
 Relational data here means anything roughly "rectangular" - with
-columns and rows.  For example, CSV it supports reading and writing
-data in:
+columns and rows.  For example, it supports reading and writing from:
 
 * Databases, including using native high-speed methods of
-  import/export of bulk data.  Redshift and Vertica are
-  well-supported, with some support for BigQuery and PostgreSQL.
+  import/export of bulk data.  Redshift, Vertica and PostgreSQL are
+  well-supported, with some support for BigQuery.
+* CSV files
+* Parquet files (initial support)
 * Google Sheets
 * Pandas DataFrames
-* CSV files, either alone or in a records directory - a structured
-  directory of CSV/Parquet/etc files containing some JSON metadata
-  about their format and origins.  Records directories are especially
-  helpful for the ever-ambiguous CSV format, where they solve the
-  problem of 'hey, this may be a CSV - but what's the schema?  What's
-  the format of the CSV itself?  How is it escaped?'
+* Records directories - a structured directory of CSV/Parquet/etc
+  files containing some JSON metadata about their format and origins.
+  Records directories are especially helpful for the ever-ambiguous
+  CSV format, where they solve the problem of 'hey, this may be a
+  CSV - but what's the schema?  What's the format of the CSV itself?
+  How is it escaped?'
 
-The record mover can be exended expand to handle additional database
-and data file types by building on top of their
-[SQLAlchemy](https://www.sqlalchemy.org/) drivers, and is able to
-auto-negotiate the most efficient way of moving data from one to the
-other.
+Records mover can be exended expand to handle additional databases
+and data file types.  Databases are supported by building on top of
+their [SQLAlchemy](https://www.sqlalchemy.org/) drivers.  Records
+mover is able to auto-negotiate the most efficient way of moving data
+from one to the other.
 
-Example CLI use:
+## Example CLI use
+
+Installing:
+
+```sh
+pip3 install 'records_mover[cli,postgres,redshift]'
+```
+
+Loading a CSV into a database:
+
+TODO
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker